src/helpers/vyos_net_name

```diff
@@ -95,11 +89,12 @@ def get_biosdevname(ifname: str) -> str:
     try:
         biosname = cmd(f'/sbin/biosdevname --policy all_ethN -i {ifname}')
```

Comment:
Should we add the missing  `--allow-vm` argument here, if we are using biosdevname for now?

Reply:
IIRC this is the only reason we keep a fork of biosdevname? If so, we should add it.